docs/sources/configure/integrations/references/manual/index.md

```diff
@@ -99,3 +99,7 @@ Navigate to the **Integrations** page and find the "Direct paging" integration f
 integration's detail page, you can customize its settings, link it to an escalation chain, and configure associated
 ChatOps channels. To confirm that the integration is functioning as intended, [create a new alert group](#page-a-team)
 and select the same team for a test run.
+
+### Important escalations
+
+TODO:
```

Comment:
will be done in a separate PR focused on auto-creating 2 routes w/ templates for Direct Paging integrations.

That PR will be merged into this branch, before merging to `dev`.